### ISSUE TITLE
Having two filters in SlickGrid

### ISSUE THREAD

**Issue by ashumahajan01** [mleibman/SlickGrid#1155]
Hello, I want to create two filters in one SlickGrid Table. One will have "input field" based filter where i should be able to put expression, i.e. value > 10 which will give me the results greater than 10 values. The second filter will be the excel like filter where it should be able to list all the values from the first filter result and once user select some values it should show those selected values as result. 
Basically, looking for combining result of two filters. 

I do have some examples as mentions below 
http://mleibman.github.io/SlickGrid/examples/example-header-row.html
http://danny-sg.github.io/slickgrid-spreadsheet-plugins/examples/example-2-everything.htm

Looking into combining above two filters in one table.
Is it possible to write two filters in one table using SlickGrid?  Let me know how to achieve the same.

**Comment by 6pac** [comment]
It's certainly possible. When filtering, you pass in a filter function that may be as complex as you like, but must return true (in result set) or false (not in result set) as a result.
So you need to build a filter function in Javascript to achieve what you want. Parsing expressions as you have described is not trivial, and you should probably use another library or tool for that. This is not the place to be asking how to program javascript stuff that is beyond your current skill level - perhaps StackOverflow is a better place.
http://6pac.github.io/SlickGrid/examples/example4-model.html is a better example to start from.
Also, you should be using [my repo](https://github.com/6pac/SlickGrid) - it's an up to date and much better patched version of this one.

**Comment by avi2911** [comment]
Hi,
Can i get an example of having two text search based external filters.

When trying to do that everything in the table gets updated based on original data instead of current data so for example,when i type 12 in a standard column it shows all the related record to that similarly when i type xxxx in name column all the related records are showing instead of showing records which are sorted by 12.

Can i get asap help in this.?

**Comment by 6pac** [comment]
The above comment is very relevant. Specifically, (1) this repo is essentially dead, and (2) please ask on StackOverflow.
